api/views.py

```diff
@@ -254,6 +260,7 @@ def get_queryset(self):
             Response.objects.filter(
                 Q(study__id__in=study_ids) | Q(child__id__in=children_ids)
             )
+            .filter(completed_consent_frame=not updating_session)
```

Comment:
Hmm - just reading code, haven't tested this. Does this mean that if we're not updating session, we require completed_consent_frame=True, and if we ARE updating session, we require completed_consent_frame=False? The latter is NOT something we want, because people need to be able to continue issuing PATCH requests as the study continues after consent. Basically the filter should just only be applied in case we're not updating the session.

Reply:
Resolving this per discussion, but would like one more pass-over for review!